test/g_searcher_web/controllers/api/search_result_controller_test.exs

```diff
@@ -60,6 +93,34 @@ defmodule GSearcherWeb.API.SearchResultControllerTest do
 
       assert json_response(conn, 200) == %{"data" => []}
     end
+
+    test "renders search results belonging to user given invalid params", %{conn: conn} do
```

Comment:
Should we update the test description here?
```suggestion
    test "renders 400 error response given invalid params", %{conn: conn} do
```

Reply:
Nice catch 😄 , I've updated the test description 9f4824ba7359218a1d49bd61f6247a36d8c2c045. 🕺